model_compression_toolkit/core/common/quantization/quantization_analyzer.py

```diff
@@ -1,71 +0,0 @@
-# Copyright 2021 Sony Semiconductor Israel, Inc. All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-# ==============================================================================
-
-from typing import Callable
-
-from networkx.algorithms.dag import topological_sort
-
-from model_compression_toolkit.core import common
-
-
-def create_tensor2node(graph: common.Graph,
-                       node: common.BaseNode,
-                       fw_info: common.FrameworkInfo):
-    """
-    Force tensor creation and assignment for a node.
-    Args:
-        graph: Graph of the node (for retrieving the current tensor).
-        node: Node to create a tensor for.
-        fw_info: Specific framework information (for example, output channels index).
-
-    """
-    current_tensor = graph.get_out_stats_collector(node)
-    is_list_nostat_collectors = isinstance(current_tensor, list) and len([sc for sc in current_tensor if not isinstance(sc, common.NoStatsCollector)]) == 0
-    if isinstance(current_tensor, common.NoStatsCollector) or current_tensor is None or is_list_nostat_collectors:
-        out_channel_axis = fw_info.out_channel_axis_mapping.get(node.type)
-        graph.set_out_stats_collector_to_node(node, common.StatsCollector(out_channel_axis))
-
-
-def analyzer_graph(node_analyze_func: Callable,
```

Comment:
why did you removed this?

Reply:
This "analyer" only sets statistic collectors to nodes. I moved it from "quantization" to model collector.